servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -221,7 +223,8 @@ public Single<? extends FilterableReservedStreamingHttpConnection> reserveConnec
         @Override
         protected Single<StreamingHttpResponse> request(final StreamingHttpRequester delegate,
                                                         final StreamingHttpRequest request) {
-            Single<StreamingHttpResponse> single = delegate.request(request);
+            final StreamingHttpRequest duplicatedRequest = request.transformMessageBody(messageBodyDuplicator());
```

Comment:
Because most users don't customize default retry strategy, we don't need to transform the body by default. Consider applying duplicator only when users customized any of `retry*` functions on the builder.